platform/testing/liquid-templates/README.md

```diff
@@ -164,6 +164,21 @@ The liquid template for the smoke test is located at `src/site/layouts/liquid_te
 
 The spec file and fixture is in the `src/site/layouts/tests/liquid_template_axe_check_smoke_test` directory.
 
+### DOM Testing Library
+
+To perform queries using the [DOM Testing Library](https://testing-library.com/), simply import the queries or functions you want to use. Example:
+```
+import { getByText } from '@testing-library/dom';
+```
+
+From the [Dom Testing Library docs](https://testing-library.com/docs/queries/about): "All of the queries exported by DOM Testing Library accept a container as the first argument."
+
+In the Liquid Template Testing Framework examples found below, the `HTML` document generated is assigned to a variable called `container` in the `spec.js` files. Use this variable as the first argument to the imported DOM Testing Library functions, like this:
+
+```
+const node = getByText(container, '3500 Ludington Street');
+```
+
```

Comment:
It might be worth adding a line or two about the difference between the `getBy...` and `queryBy...` queries. 

When I first learned about them, I lost hours trying to assert that something was absent because I was using the wrong query. 

https://testing-library.com/docs/queries/about#types-of-queries

>- `getBy...`: Returns the matching node for a query, and `throw` a descriptive error if no elements match or if more than one match is found (use `getAllBy` instead if more than one element is expected).
>- `queryBy...`: Returns the matching node for a query, and return `null` if no elements match. **This is useful for asserting an element that is not present.** Throws an error if more than one match is found (use `queryAllBy` instead if this is OK). 
```

Reply:
Thanks for the comment. I don’t think I want to include that information in this doc. Perhaps we can create a doc about the Testing Library itself at some point which would be a great place for tips and tricks, etc…